.pipelines/android_packaging.yml

```diff
@@ -0,0 +1,78 @@
+# packaging pipeline for onnxruntime-extensions Android AAR package
+
+parameters:
+  - name: IsReleaseBuild
+    displayName: "Is this is a release build?"
+    type: boolean
+    default: false
+
+name: "$(Date:yyyyMMdd)$(Rev:rrr)" # build number format
+
+jobs:
+  - job: AndroidPackaging
+    pool:
+      vmImage: "macOS-12"
+    timeoutInMinutes: 120
+    variables:
+      OnnxruntimeDirectory: cmake/external/onnxruntime
+    steps:
+      - script: brew install coreutils ninja
+        displayName: Install coreutils and ninja
+
+      - script: |
+          mkdir cmake/external
+          cd cmake/external
+          git clone https://github.com/microsoft/onnxruntime.git
+          cd onnxruntime
+          git checkout v1.12.1
+        displayName: Download Onnxruntime
+        workingDirectory: $(Build.sourcesdirectory)
+
+      - script: |
+          python3 tools/python/run_android_emulator.py \
+            --android-sdk-root ${ANDROID_SDK_ROOT} \
+            --create-avd --system-image "system-images;android-30;google_apis;x86_64" \
+            --start --emulator-extra-args="-partition-size 4096" \
+            --emulator-pid-file $(Build.BinariesDirectory)/emulator.pid
+        displayName: Start Android emulator
+        workingDirectory: $(OnnxruntimeDirectory)
+
+      - task: JavaToolInstaller@0
+        displayName: Use jdk 11
+        inputs:
+          versionSpec: "11"
+          jdkArchitectureOption: "x64"
+          jdkSourceOption: "PreInstalled"
+
+      # TODO: set package version number here
+
+      - script: |
+          bash ./tools/android.package.sh
```

Comment:
should we specify the ops for the package?
e.g.:
https://github.com/microsoft/onnxruntime-extensions/blob/56ef48073c4e58c44c69f27e9b5fb7c8cf29b7f4/.pipelines/ios_packaging.yml#L30-L32
and then set OCOS_ENABLE_SELECTED_OPLIST=ON

however, I think the android test uses ops which aren't currently in the ios package ops list

Reply:
Yes, thanks for the info. I can add an android/package_ops.config file. The android test currently uses BertTokenizer op.

Do you suggest me to modify the android test or what's the requirement for the package ops list? is it fine to just include BertTokenizer?

Reply:
I think we should include the EncodeImage and DecodeImage ops in the config. it's probably fine to also include the BertTokenizer op for now if that is easier.